tests/components/lamarzocco/test_config_flow.py

```diff
@@ -272,6 +273,69 @@ async def test_reauth_flow(
     assert mock_config_entry.data[CONF_PASSWORD] == "new_password"
 
 
+async def test_reconfigure_flow(
+    hass: HomeAssistant,
+    mock_cloud_client: MagicMock,
+    mock_config_entry: MockConfigEntry,
+    mock_device_info: LaMarzoccoDeviceInfo,
+) -> None:
+    """Testing reconfgure flow."""
+    mock_config_entry.add_to_hass(hass)
+
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN,
+        context={
+            "source": SOURCE_RECONFIGURE,
+            "unique_id": mock_config_entry.unique_id,
+            "entry_id": mock_config_entry.entry_id,
+        },
+        data=mock_config_entry.data,
+    )
+
+    assert result["type"] is FlowResultType.FORM
+    assert result["step_id"] == "reconfigure_confirm"
+
+    result2 = await __do_successful_user_step(hass, result, mock_cloud_client)
+    service_info = get_bluetooth_service_info(
+        mock_device_info.model, mock_device_info.serial_number
+    )
+
+    with (
+        patch(
+            "homeassistant.components.lamarzocco.config_flow.LaMarzoccoLocalClient.validate_connection",
+            return_value=True,
+        ),
+        patch(
+            "homeassistant.components.lamarzocco.config_flow.async_discovered_service_info",
+            return_value=[service_info],
+        ),
```

Comment:
I thought it was possible to inject discovered services

Reply:
not that I know of